lib/mix/lib/mix/tasks/test.ex

```diff
@@ -486,7 +486,7 @@ defmodule Mix.Tasks.Test do
     cover =
       if opts[:cover] do
         compile_path = Mix.Project.compile_path(project)
-        partition = opts[:partitions] && System.get_env("MIX_TEST_PARTITION")
+        partition = opts[:partitions] > 1 && System.get_env("MIX_TEST_PARTITION")
```

Comment:
Maybe we should Delete the option if it is set to 1 so we don’t have to handle it elsewhere?